packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -114,7 +114,7 @@ interface OpLatencyMenuSelection {
  * E.g. {@link ContainerMenuSelection} represents that the menu option for a Container
  * is selected, and has a 'containerKey' property to indicate which Container.
  */
-type MenuSelection =
+export type MenuSelection =
```

Comment:
This whole type union is duplicated here and in src/components/Menu.tsx, I think only the latter should exist. Can we clean it up here too?